extensions/wikia/TemplateClassification/scripts/TemplateClassification.js

```diff
@@ -0,0 +1,12 @@
+require(['jquery'], function ($) {
+	'use strict';
+
+	function init() {
+		$('.template-classification-edit').click(function (e) {
+			e.preventDefault();
+			console.log('edit');
```

Comment:
http://stackoverflow.com/questions/8002116/should-i-be-removing-console-log-from-production-code

Use a comment to mark the method as a stub. :)